src/tests/e2etest/E2ETest.cs

```diff
@@ -24,7 +24,7 @@ public class E2eTest
         private string deviceId = Environment.GetEnvironmentVariable("E2E_OSCONFIG_DEVICE_ID");
         private readonly string moduleId = "osconfig";
         protected int twinTimeoutSeconds = 0;
-        private int twinTimeoutSecondsDefault = 45;
+        private int twinTimeoutSecondsDefault = 180;

```

Comment:
Not sure i agree with bumping the global timeout. The timeout should be sufficient given the reporting interval on the tests is 30s, twin updates will occur before this. I suspect something else is at play here if it is necessary to bump the timeout. Maybe this was due to the stale twin when checking for the `WaitForCommandStatus` call I commented above?

Reply:
These tests must continue to run w/o failure if we change the default reporting interval in the future. The tests should be agnostic to that.

Reply:
yeah good point. I agree it should work for all intervals, not sure how we can adjust for that though but as far as our e2etests are concerned, we always configure our agents to use 30s. I just mean to say here that having to bump the timeout for the tests to work is concerning, we shouldn't have to do this

Reply:
Bumped the timeout as per an earlier comment. When tests are refactored and tooling is added to aid E2E test development, we should be able to reduce this/abstract it away from the tests entirely.